### ISSUE TITLE
No output from printer in 5.Distributed System

### ISSUE THREAD

**Issue by kehusa** [HEADS-project/training#19]
System is successfully deployed (I think), but no ticker is printed in the JSNode console:¨

![image](https://cloud.githubusercontent.com/assets/8616423/5628786/183b4f26-95ac-11e4-8d6a-f2c79f48522c.png)

JSNode: 
![image](https://cloud.githubusercontent.com/assets/8616423/5628824/8d0f92a8-95ac-11e4-8abd-93c365ba0a67.png)

**Comment by maxleiko** [comment]
Probably a bug introduced by the changes in Kevoree 5.2.1 and the new libraries (v5.2.2).  
I'm digging into it as I write that :)

**Comment by brice-morin** [comment]
I have somehow a similar problem, with some more outputs in the console.

In JS:

```
09:07:06  DEBUG  KevoreeCore      Model deployed successfully: 14 adaptations (1299ms)
09:07:06  INFO   WSMsgBroker      "broker" + /nodes[node0]/components[printer]/provided[input]_chan
09:07:06  INFO   RuntimeGruntTa.  /nodes[node0]/components[printer]/provided[input]_chan registered on remote server
09:07:08  INFO   WSMsgBroker      "broker" + /nodes[javaNode]/components[ticker]/required[tick]
09:07:11  WARN   WSMsgBroker      "broker" broker warning: Unable to send message. Client "/nodes[node0]/components[prin
ter]/provided[input]" unknown
09:07:14  WARN   WSMsgBroker      "broker" broker warning: Unable to send message. Client "/nodes[node0]/components[prin
ter]/provided[input]" unknown
09:07:17  WARN   WSMsgBroker      "broker" broker warning: Unable to send message. Client "/nodes[node0]/components[prin
ter]/provided[input]" unknown
09:07:20  WARN   WSMsgBroker      "broker" broker warning: Unable to send message. Client "/nodes[node0]/components[prin
ter]/provided[input]" unknown
09:07:23  WARN   WSMsgBroker      "broker" broker warning: Unable to send message. Client "/nodes[node0]/components[prin
ter]/provided[input]" unknown
09:07:24  WARN   NodeJSRuntime    Got SIGINT.  Shutting down Kevoree gracefully... (^C again to force quit)
```

In Java:
```
01:07 INFO: JavaNode Update completed in 27 ms
01:07 INFO: End deploy result=true-27
01:07 INFO: WSGroup "group" update result: true
01:10 WARN: Unable to send message, no connection established for /nodes[javaNode]/components[ticker]/required[tick]
01:28 WARN: Unable to send message, no connection established for /nodes[javaNode]/components[ticker]/required[tick]
01:28 INFO: Stopping Kevoree
```

**Comment by maxleiko** [comment]
Fixed fa90b99ac3599e6494626e7d43d40c7cc38b5124

**Comment by brice-morin** [comment]
Indeed :-)
